documentation/docs/steps/transportRequestReqIDFromGit.md

```diff
@@ -14,6 +14,18 @@ See [transportRequestUploadSOLMAN](transportRequestUploadSOLMAN.md).
 With `transportRequestReqIDFromGit`  you can retrieve the transport request ID from the commit message of the Git repository of your project. This way, you can address the transport request without having to change the setup of your pipeline.
 Please make sure that the ID is unique in the defined search range.
 
+## General purpose pipeline Init Stage
+
+The step can also be configured via General purpose pipeline in Init stage using config.yml. The configuration will look as below:
+
+```yaml
+stages:
+  Init:
+    transportRequestReqIDFromGit: true
+```
```

Comment:
If step config exists for this step, above initialization is not needed in my opinion.

Reply:
As this step can function without configuration, this is necessary, IMO.